resources/knative-eventing/charts/knative-eventing/templates/eventing.yaml

```diff
@@ -126,16 +116,7 @@ spec:
 ---
 apiVersion: v1
 data:
-  default-channel-config: |
-    clusterdefault:
-      apiversion: eventing.knative.dev/v1alpha1
-      kind: ClusterChannelProvisioner
-      name: in-memory
-    namespacedefaults:
-      some-namespace:
-        apiversion: eventing.knative.dev/v1alpha1
-        kind: ClusterChannelProvisioner
-        name: some-other-provisioner
+  default-channel-config: ""
```

Comment:
Any particular reason to do that? Any way You should mention the change in README